homeassistant/components/http/ban.py

```diff
@@ -89,7 +89,9 @@ def log_invalid_auth(func):
     remote_addr = request[KEY_REAL_IP]
 
     msg = ('Login attempt or request with invalid authentication '
-           'from {}'.format(remote_addr))
+           'from {}. Method: {}, Path: {}'.format(remote_addr,
+                                                  request.method,
+                                                  request.raw_path))
```

Comment:
Password, API key and access tokens must be masked or not printed in the log output. We don't want to  deal with the leaking of private information in the forum and in the issues again.

Reply:
I swapped out `raw_path` to `path` which should remove the security issue with the `api_password`

See updated description